DNS/Server/MasterFile.cs

```diff
@@ -44,6 +44,12 @@ public class MasterFile : IRequestResolver {
             entries.Add(entry);
         }
 
+		public void AddSrvRecord(string service, string target, int port, int priority, int weight)
```

Comment:
The formatting here still looks weird. Can you also fix the brackets, so that the opening bracket is not on a newline, like the rest of the code.

Reply:
Sweet